.clusterfuzzlite/build.sh

```diff
@@ -0,0 +1 @@
+compile_go_fuzzer github.com/decred/dcrd/crypto/blake256 FuzzSum256 fuzz_sum256
```

Comment:
Then adding the build tag here:

```suggestion
compile_go_fuzzer github.com/decred/dcrd/crypto/blake256 FuzzSum256 fuzz_sum256 fuzz
```